projboard/tests/test_login.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from django.contrib.auth.models import User
+from pytest_django.asserts import assertTemplateUsed
+from conftest import (VALID_USER, VALID_EMAIL, VALID_PASSWORD, INVALID_PASSWORD_MISMATCH, LOGIN_PATH)
+
+
+@pytest.mark.django_db
+class TestLogin:
+    @pytest.fixture
+    def registered_user(self):
+        user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
+        user.save()
```

Comment:
Maybe we can use this fixture in the future, consider to move it to `conftest.py`.

Reply:
+1 you might need it in other tests

Reply:
It is no longer relevant